CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [CHANGE] Ruler: Add `user` and `reason` labels to `cortex_ruler_write_requests_failed_total` and `cortex_ruler_queries_failed_total`; add `user` to
     `cortex_ruler_write_requests_total` and `cortex_ruler_queries_total` metrics. #10536
 * [FEATURE] Distributor: Add experimental Influx handler. #10153
+* [FEATURE] Helm: Add support for `.service.extraPorts` on all components. #10659
```

Comment:
The Helm cart has its own changelog in the [`operations/helm/charts/mimir-distributed/CHANGELOG.md`](https://github.com/grafana/mimir/blob/03e9612ac35d709c1e8678423e9b3e105379dca3/operations/helm/charts/mimir-distributed/CHANGELOG.md). Please, move this one there.

Also, it would be great if the changelog could add some context about the change, e.g.

```suggestion
[ENHANCEMENT] All componentes can expose additional ports with their respective services via the *.service.extraPorts values. This allows exposing the containers defined in a component's *.extraContainers. #10659
```

Reply:
Updated with your syntax.  Nice suggestion, thanks.